_posts/2015-08-01-Why-you-should-always-follow-your-dreams.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+name: Why-you-should-always-follow-your-dreams!
+title: Why you should always follow your dreams!
+date: 2015-08-01 12:00:00 +00:00
+author: Ana Nogal
+image:
+    src: /assets/img/custom/blog/follow-your-dreams.jpg
+    attribution:
+        text: Follow your dreams
+tags:
+- dreams
+- codurance
+- carrer
+- culture
+---
+I started at [Codurance](http://codurance.com/) a month and a half ago  as a C# developer. I was glad to have the opportunity to work in a place, where people value your effort to be a better developer. They set you some goals that you are expect to achieve on your own, with their support.
+
+I was happy with my position, I had been away from C# for more than 4 years. This was for me a great opportunity to catch up with my main language. But my passion is with iOS. On my own time, I started to apply all the techniques that I was learning in C# to iOS.  You can have a look [here](http://wp.me/p4i8Xl-6y) or [here](http://codurance.com/2015/07/13/How-to-start-doing-TDD-in-iOS/) were I share my progress.
+
+And you think: Why did you apply for C# and not tried iOS? Well, I looked for 3 months for a iOS company that could give me an opportunity. It was really hard. They all wanted senior developers who could get there and got the job done. But I'm not at that level in iOS. I still have a lot to learn and I needed an opportunity to learn more from experienced developers. Exchange thoughts and understand how things work. I had that in C#, I've been in several clients, I knew how to address a problem, how I could put my expertise to help. That's not true in iOS. And that's what I missed from my journey in the world of apps. To have this kind off experience. And you thinking: didn't you develop 4 apps? Didn't you pass for all the desperate days of deployment for the AppStore? Yes, I did! But, I was my client, and the developer. It started like a small pet project that got big enough for me to stay at home for 4 years and try to make it work.  I missed the experience exchange  in a team. The great discussions that a subject can lead to... This is what makes you grow in your profession.
```

Comment:
> And you thinking: didn't you develop 4 apps? Didn't you pass for all the desperate days of deployment for the AppStore?

I don't think most readers will know this about you. You should go into more detail. :smiley: